workflow.rst

```diff
@@ -596,6 +596,19 @@ transition. The value of this option is any valid expression created with the
             ],
         ]);
 
+.. note::
+
+    If `symfony/security-core` package is not installed, these methods are not available:
```

Comment:
These expressions instead of methods, wdty?

Reply:
I changed `methods` to `expressions`, please take a look